charts/kamus/README.md

```diff
@@ -59,6 +59,7 @@ The chart can be customized using the following configurable parameters. Most se
 | `keyManagement.awsKms.region`                | AWS KMS region   | 
 | `keyManagement.awsKms.cmkPrefix`                | Prefix for the customer master keys that are created in the KMS  | 
 | `pod.annotations`                             | Set the annotations for Kamus's pods | 
+| `crdConersationEnabled`                       | The `CustomResourceWebhookConversion` is enabled (relevant for clusters bellow version 1.15.0 where it's enabled by default). Required for supporting conversations of CRDs objects from older to new version. |
```

Comment:
Can you explain a bit about this value? why under 1.15 it's opt-in feature?

Reply:
I tried to make it clearer - take another look, please? Under 1.15 the CRD conversion webhook is disabled by default.